### PR TITLE
Make small improvement to uprading.md

### PR DESCRIPTION
**Pull request by toureholder**
Changes proposed in this pull request:

*  Just a small correction to the Upgrading the Flutter SDK docs.

### CODE REVIEW

File: src/docs/development/tools/sdk/upgrading.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Flutter SDK or the packages that your app depends on.
 
 ## Upgrading the Flutter SDK
 
-To update both the Flutter SDK use the `flutter upgrade` command:
+To update both the Flutter SDK and the packages that your app depends on, use the `flutter upgrade` command:
```
Comment (sfshaza2) [inline src/docs/development/tools/sdk/upgrading.md:13]: "flutter upgrade" upgrades the Flutter SDK, but it doesn't upgrade your pub dependencies. I am not quite sure what you mean by this change.

**Comment by sfshaza2** [comment]
Closing.

File: src/docs/development/tools/sdk/upgrading.md (hunk 1 above)
Comment (toureholder) [reply src/docs/development/tools/sdk/upgrading.md:13]: Ok @sfshaza2! I guess, it's the "both" that should be removed then. The phrase, should be "To update the Flutter SDK use..." (with out the "both"). "Both" what? Get me?

**Comment by sfshaza2** [comment]
You are 100% right! That "both" should be deleted.

**Comment by toureholder** [comment]
> You are 100% right! That "both" should be deleted.

I updated the PR @sfshaza2

**Review by sfshaza2 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**